src/scss/variables.scss

```diff
@@ -12,7 +12,8 @@ $alert-secondary-bg-color: #e2e3e5;
 
 // Shared style for the CodeView and DiffView components.
 $code-font-family: Consolas, Courier, monospace;
-$code-font-size: 16px;
+$code-font-size: 13px;
```

Comment:
Could we use `em` here given that the body font-size is going to be a percentage value?

Reply:
`13px` seems important for viewing code. What would be better about using a percentage? Just for consistency with the rest of the site?

Reply:
Yep. If one wants to increase the font (using the browser zoom feature), then the font size of the code should also be increased.

Reply:
Browsers now scale pixel sized fonts when zooming. Is there any other reason not to use pixels?

Reply:
If so, then let's switch to pixels for everything then? Including the `body` font size, WDYT?

Reply:
Yeah, good idea.